docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '2'
+services:
+  web:
+    image: "myimage"
```

Comment:
@blackavgn hm, i don't think that it my image. maybe yours? (sarcasm). Please use normal name